### ISSUE TITLE
 How to get stable feature loading of CP decomposition？

### ISSUE THREAD

**Issue by wanglu2014** [tensorly/tensorly#384]
Even run with the same input array and parameters, the factors of parafac result would be changed. How to get robust factors?

**Comment by aarmey** [comment]
Indeed, this is a challenge with CP decomposition. This arises from the fact that ALS is dependent on a starting point, and so it becomes important to have a good estimate of the resulting factors at the start. It is perfectly acceptable to run the method multiple times and take the best-fitting result. SVD initialization can also help you get to a better result as well. Keep in mind that CP tensors are scaling, sign, and permutation invariant, and so results that seemingly look different might in fact be essentially the same.

By the way, we have a [Slack channel](https://join.slack.com/t/newworkspace-bh44369/shared_invite/zt-u21984bp-O7jOb3pSOr4hzMLoDME0wA), where it might be easier for quick questions and interaction.

**Comment by wanglu2014** [comment]
Dear Dr Meyer, thank you for your kindly reply. However, the Slack channel page said that it need a new invitation. Could you kindly sent an invitation to my email cs001632@gmail.com. Thank you for your help!

**Comment by JeanKossaifi** [comment]
Hi @wanglu2014  - here's an [updated link for Slack](https://join.slack.com/t/tensorly/shared_invite/zt-wqnts2sk-wbiRX6ml~Xt6~GDYWRPFfg)

**Comment by wanglu2014** [comment]
> Hi @wanglu2014 - here's an [updated link for Slack](https://join.slack.com/t/tensorly/shared_invite/zt-wqnts2sk-wbiRX6ml~Xt6~GDYWRPFfg)

I have joined the channel. Thank you for your timely reply!

**Comment by JeanKossaifi** [comment]
@wanglu2014 @aarmey should we close this or create a PR with more doc/user-guide to address this question?

**Comment by aarmey** [comment]
It would be helpful to have some information about this in the docs somewhere. This comes up all the time.

**Comment by wanglu2014** [comment]
> @wanglu2014 @aarmey should we close this or create a PR with more doc/user-guide to address this question?

I receive no related advice from Slack, and have decided to perform analysis as you sugguested in Mar 9.
